src/DotNetCore.CAP.AzureServiceBus/Options/AzureServiceBusOptionsBase.cs

```diff
@@ -7,42 +7,22 @@
 using Azure.Core;
 using Azure.Messaging.ServiceBus;
 using DotNetCore.CAP.AzureServiceBus;
-using DotNetCore.CAP.AzureServiceBus.Producer;
 
 // ReSharper disable once CheckNamespace
 namespace DotNetCore.CAP;
 
 /// <summary>
 /// Provides programmatic configuration for the CAP Azure Service Bus project.
 /// </summary>
-public class AzureServiceBusOptions
+public class AzureServiceBusOptionsBase
```

Comment:
What's the purpose of separating this in a base class?

Reply:
Because I reuse this option every custom consumer. And AzureServiceBusOptions is extened AzureServiceBusOptionsBase for defaul options